### DISCUSSION TITLE
AudioBaseInfo, AudioInfo and improvements

### DISCUSSION THREAD

**Discussion by pschatzmann** [pschatzmann/arduino-audio-tools#746] [Announcements]
I decided to rename the AudioBaseInfo to simply AudioInfo which is more natural!

I also noticed that many sketches contain the sample rate, channels and bits per sample as global variables in the top and in the sketch create an AudioBaseInfo (or new AudioInfo) and assign these values.

So for this old way of doing things 
```C++
uint16_t sample_rate=44100;
uint8_t channels = 2;    
uint8_t bits_per_sample = 16;                                  
AudioInfo info;
info.sample_rate = sample_rate
info.channels = channels
info.bits_per_sample = bits_per_sample
```
Now we can just create a global AudioInfo object and assign the values in the constructor

```C++
AudioInfo info(44100, 2, 16);
```

**Comment by aqildad-create** [comment]
very appreciated .
